modules/uploading-an-existing-notebook-file-from-a-git-repository-using-code-server.adoc

```diff
@@ -0,0 +1,30 @@
+:_module-type: PROCEDURE
+
+[id='uploading-an-existing-notebook-file-from-a-git-repository-using-code-server_{context}']
+= Uploading an existing notebook file from a Git repository by using code-server
+
+[role='_abstract']
+You can use the code-server user interface to clone a Git repository into your workspace to continue your work or integrate files from an external project.
+
+.Prerequisites
+* A launched and running code-server workbench.
+* Read access for the Git repository you want to clone.
+
+.Procedure
+. Copy the HTTPS URL for the Git repository.
++
+** On GitHub, click *&#10515; Code* -> *HTTPS* and click the Copy URL to clipboard icon.
```

Comment:
- Change "On GitHub" to "In GitHub"?
- apply bold font to "Copy URL to clipboard"?

Reply:
> Change "On GitHub" to "In GitHub"?

interesting suggestion, I'd say on, because it's a website and not an app (in my perception)